docs/api/spec/components/schemas/ipset.yaml

```diff
@@ -0,0 +1,48 @@
+IpsetNetwork:
+    type: object
+    properties:
+        network:
+            type: string
+            description: The IPv4 address of the network
+
+IpsetRoleId:
+    type: object
+    properties:
+        role_id:
+            type: integer
+            description: The role of the clients node
+
+IpsetIp:
+    type: object
+    properties:
+        ip:
+            type: string
+            description: The IPv4 address of the client
+
+IpsetPort:
+    type: object
+    properties:
+        port:
+            type: string
+            description: The port type:integer example: tcp:65535
```

Comment:
"The port to allow through. Can be prefixed by `tcp:` or `udp:`. When the protocol is omited, then the port will be opened for both protocols"